### PR TITLE
docs(release): Adding troubleshooting section - TWIG-142

### PR DESCRIPTION
**Pull request by planctus**
# PR description

Please drop a few lines about the PR: what it does, how to test it, etc.

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

- [ ] I have put the vanilla component as `devDependencies`
- [ ] I have put the specs package as `devDependencies`
- [ ] I have added the components directly used in the twig file (with `include` or `embed`) as `dependencies`
- [ ] My component is listed in `@ecl-twig/ec-components`'s `dependencies`
- [ ] My variables naming follow the guidelines (snake case for twig)
- [ ] I have provided tests
- [ ] I have provided documentation (for the "notes" tab)
- [ ] If my local `yarn.lock` contains changes, I have committed it
- [ ] I have given my PR the proper label (`pr: review needed` to indicate that I'm done and now waiting for a review ,`pr: wip` to indicate that I'm actively working on it ...)

### CODE REVIEW

File: docs/release.md [hunk 1]
```diff
@@ -57,3 +57,27 @@ Note: you might be bombarded with emails from npm (one for each published packag
 Final step: create a new GitHub release from the `master` branch. The tag version should be prefixed with `v`, e.g. `v2.3.0`. As a title, you can use something like `2.3.0 (2019-06-14)` (like in the root `CHANGELOG.md` but without the link). And for the description of the release, you can copy-paste the entry generated in the root `CHANGELOG.md` as well (I know, we're duplicating a lot of information...).
 
 Now, enjoy a well-deserved break! :wink:
+
+
+## Troubleshooting
+
+We experienced some failures while publishing on npm sometimes and we could not fully understand the reason for those but we could, at least, find a procedure that worked in these cases.
+The result of the *npm run publish* command was a partial publication of the packages in the release (it happened with a different number of packages correctly published) and after the script stopped returning an error we had all the packages.json files modified in the various components with the addition of a "githead" property with an hash.
```
Comment (kalinchernev) [inline docs/release.md]:
```suggestion
In case of failure during publishing packages, retry to publish missing packages after clearing history:
```

**Comment by kalinchernev** [comment]
In my opinion, https://github.com/ec-europa/ecl-twig/pull/237/files#diff-8c625b93dfb06ed7422ad4097ead757cR64-R69 is still too much information, just sharing.

**Comment by planctus** [comment]
I agree @kalinchernev, reduced the amount of text. ;)

**Comment by kalinchernev** [comment]
@planctus thanks for the updates!

**Pull request merged**